File: MHealthPlus/src/com/lucentinsight/mclinicplus/data/Doctor.java
package com.lucentinsight.mclinicplus.data;

import java.util.ArrayList;
import java.util.List;

public class Doctor {

	private String name;
	private String qualification;
	private String specialization;
	private List<Schedule> schdules=new ArrayList<Schedule>();
	
	
	public List<Schedule> getSchdules() {
		return schdules;
	}
	public void setSchdules(List<Schedule> schdules) {
		this.schdules = schdules;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getQualification() {
		return qualification;
	}
	public void setQualification(String qualification) {
		this.qualification = qualification;
	}
	public String getSpecialization() {
		return specialization;
	}
	public void setSpecialization(String specialization) {
		this.specialization = specialization;
	}
	
	public void addSchedule(Schedule schedule){
		schdules.add(schedule);
	}
}

File: backend/mclincBackend/mclinicplus.com/php/entities/Doctor.php
<?php
/** @Entity
  * @Table(name="CP_DOCTOR")
*/
class Doctor
{
/** @Id @Column(type="integer")
 * @GeneratedValue
 */
   private $id;

      public function getId(){
      return $this->id;
      }

	  public function setId($id){
	   $this->id=$id;
	   }




   /** @Column(type="string") */
      private $name;

      public function getName(){
	        return $this->name;
	        }

	  	  public function setName($name){
	  	   $this->name=$name;
	   }



	  /** @Column(type="string") */
      private $qualification;

      public function getQualification(){
	        return $this->qualification;
	  }

	  public function setQualification($qualification){
	  	   $this->qualification=$qualification;
	   }


   /** @Column(type="string") */
      private $specialization;

      public function getSpecialization(){
	  return $this->specialization;
	  }

	  public function setSpecialization($specialization){
	  $this->specialization=$specialization;
	   }

        private $schedules;

     public function addSchedule($schedule){
         $this->schedules[]=$schedule;
     }

function __construct() {
       $this->schedules=array();
   }

 public function toJSON(){
	// return '{"name":"'.$this->name;
	 $ret= '{ "name":"'.$this->name.'","qualification":"'.$this->qualification.'","specialization":"'.$this->specialization.'","schedule":[{}';
     if(count($this->schedules)!=0){
      foreach($this->schedules as $schedule){
   $ret=$ret.','.$schedule->toJSON();
 }   
     }
     $ret=$ret.']}';
     return $ret;
	 }
}
?>